### PR TITLE
Bookstore: add reducers and actions

### PR DESCRIPTION
**Pull request by veronica365**
### Description

- Includes an array of books (initial state: empty array)
- Includes a reducer that adds a book
- Includes a reducer that removes a book
- Includes an array of categories (initial state: empty array)
- Includes a reducer that checks the status and always returns "Under construction" (the initial state should check to that string)

### CODE REVIEW

File: src/redux/store.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: {
+    books: bookReducer,
+  },
+});
+export default store;
```
Comment (didierganthier) [inline src/redux/store.js]: Nice job overall 💯  but you should import `categories` and use `configureStore` to create a reducer. Also  `books` and `categories` should be added to the `configureStore` method. For now you only have `books` 🔍

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @veronica365 ,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct Gitflow ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

File: src/redux/store.js (hunk 1 above)
Comment (veronica365) [reply src/redux/store.js]: Clearly this was not necessary. Anyways thanks for the review

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @veronica365  

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights
- Descriptive PR ✔️
- Nice component usage✔️
- Correct GitHub flow ✔️

## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.
Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/redux/store.js (hunk 1 above)
Comment (didierganthier) [reply src/redux/store.js]: Not according to the project requirements @veronica365 👇🏽 

Screenshot 2023-04-05 at 07 13 51

So I'm just doing my job, please be respectful. 😢

**Pull request merged**